src/libs/ActiveClientManager/index.js

```diff
@@ -7,6 +7,16 @@ const clientID = Str.guid();
 const maxClients = 20;
 
 let activeClients;
+let setReady = null;
+
+// Whether the client Manager has started
+let isInit = false;
```

Comment:
Instead could we have something like:

`shouldFireOnReadyCallback` and `didInitialize` and then do

```js
if (shouldFireOnReadyCallback) {
    onReadyCallback();
    didInitialize = true;
}
```